build.xml

```diff
@@ -222,6 +234,9 @@ TODO consider switching back to this at 9.8.0/9.9build
     </target>
     <target name="jar" depends="test" description="creates a jar of our classes">
         <antcall target="version"/>
+
+        
+
```

Comment:
Any particular reason why three new blank lines have been added?  (I can understand adding a single one, for readability/spacing, but three new blank lines seems unnecessary to me...)

Reply:
No.  Will remove and push.

Reply:
pushed fix